contracts/BitGuildWhitelist.sol

```diff
@@ -0,0 +1,111 @@
+pragma solidity ^0.4.22;
+
+import "./BitGuildAccessAdmin.sol";
+
+
+/**
+ * @title BitGuildWhitelist
+ * @dev A small smart contract to provide whitelist functionality and storage
+ */
+contract BitGuildWhitelist is BitGuildAccessAdmin {
+    address[] public whitelist;
```

Comment:
I don't think you want it here. For games may be fine, but for pretty much everything that exceeds the count of 50 it would start to break.

Reply:
Makes sense. Removed.